dockerfiles/wazuh/data_dirs.env

```diff
@@ -0,0 +1,8 @@
+i=0
+DATA_DIRS[((i++))]="etc"
+DATA_DIRS[((i++))]="logs"
+DATA_DIRS[((i++))]="stats"
+DATA_DIRS[((i++))]="queue"
+DATA_DIRS[((i++))]="var/db"
+DATA_DIRS[((i++))]="api"
+export DATA_DIRS
```

Comment:
yes, this is OK. this script copies dirs/files listed here to host-volume. I wanted to make containers manage rulesets instead of host